src/network/middleware.es

```diff
@@ -14,43 +13,37 @@ export default (store) => (
 
       // raw network call
       let promise;
-      let result;
       if (typeof request === 'function') {
-        result = request(createApi(state.token), store.dispatch, state);
-
-        if (!result || !result.request || !result.promise) {
-          throw new Error('request function must return a hash with `request` and `promise` props');
-        }
-
-        promise = result.promise;
+        promise = request(createRequest, store.dispatch, state);
```

Comment:
why pass it? we can then import it to actions module

Reply:
Yea could prob remove this - not used